### PR TITLE
Update metallica.html

### PR DESCRIPTION
**Pull request by kristjanjansen**

### CODE REVIEW

File: metallica.html [hunk 1]
```diff
@@ -1,45 +1 @@
-<style>
```
Comment (KaroliinaPilden) [inline metallica.html]: Deleting the whole file isn't okey.

Comment (kristjanjansen) [reply metallica.html]: We need to start from scratch since ALL the bands need same style

Comment (KaroliinaPilden) [reply metallica.html]: Style can be changed, no need to delete everything(pictures, link etc.)

**Comment by kasparkivi** [comment]
fondipoiss vajab fixxi

**Pull request closed without merging**